C++/C.sublime-syntax

```diff
@@ -306,12 +303,11 @@ contexts:
         2: constant.numeric.value.c
         3: punctuation.separator.decimal.c
         4: constant.numeric.suffix.c
-        5: constant.numeric.suffix.c
-        6: invalid.illegal.numeric.suffix.c
-        7: constant.numeric.value.c
-        8: punctuation.separator.decimal.c
-        9: constant.numeric.suffix.c
-        10: invalid.illegal.numeric.suffix.c
+        5: invalid.illegal.numeric.suffix.c
+        6: constant.numeric.value.c
+        7: punctuation.separator.decimal.c
+        8: constant.numeric.suffix.c
+        9: invalid.illegal.numeric.suffix.c
```

Comment:
Without distinct double/float suffixes, we can probably simplify the pattern further 

```yml
    - match: |-
        (?x)
        (
          \b{{dec_digits}} 
          (?:
          # 1., 1.f, 1.L, 1.1, 1.1e1, 1.1e-1, 1.1f, 1.1e1f, 1.1e-1f, 1.1L, 1.1e1L, 1.1e-1L
            (\.) (?: {{dec_digit}}+ | (?!\.) ) {{dec_exponent}}?
          # 1.e1, 1.e-1, 1.e1f, 1.e-1f, 1.e1L, 1.e-1L
          | {{dec_exponent}}
          )
        # .1, .1e1, .1e-1, .1f, .1e1f, .1e-1f, .1L, .1e1L, .1e-1L
        |  (\.) {{dec_digit}}+ {{dec_exponent}}?
        )
        (?: ({{double_suffix}}) | ({{dec_suffix}}) )?
      scope: meta.number.float.decimal.c
      captures:
        1: constant.numeric.value.c
        2: punctuation.separator.decimal.c
        3: punctuation.separator.decimal.c
        4: constant.numeric.suffix.c
        5: invalid.illegal.numeric.suffix.c
```